src/router/index.js

```diff
@@ -18,6 +18,11 @@ const router = createRouter({
       path: '/dashboard',
       name: 'dashboard',
       component: () => import('../views/DashboardView.vue'),
+    },
+    {
+      path: '/dashboard/filter',
+      name: 'filter',
```

Comment:
Change route to /store, and remove /filter should show a query like ?category='cat-name'